qiskit/schemas/backend_configuration_schema.json

```diff
@@ -255,7 +255,26 @@
                                 "type": "array",
                                 "minItems": 0,
                                 "description": "A list of available parametric pulse shapes",
-                                "items": {"type": "string"}}
+                                "items": {"type": "string"}},
+                            "channels": {
+                                "type": "object",
+                                "minItems": 1,
```

Comment:
Do we have to worry about backwards compatibility? Will all backends have a `channels` or does this just mean _if_ it has `channels` it needs at least one item?

Reply:
Ahh, I hadn't caught this. It means if it has a channel it should have at least 1. However, I don't think that should be required.

Reply:
fixed in [7da1c51](https://github.com/Qiskit/qiskit-terra/pull/3904/commits/7da1c51216997f257241cecb522882877858a1e1)